.gitignore

```diff
@@ -2,3 +2,4 @@
 *.root
 *json
 *.txt
+files/*.root
```

Comment:
can you please add .pdf files to the .gitignore and remove all the PDFs added to the repository?